packages/@aws-cdk/aws-batch/lib/job-definition.ts

```diff
@@ -197,6 +235,28 @@ export interface JobDefinitionContainer {
    * @default - No data volumes will be used.
    */
   readonly volumes?: ecs.Volume[];
+
+  /**
+   * The platform configuration for jobs that are running on Fargate resources.
+   *
+   * @default - LATEST platform version will be used
+   */
+  readonly fargatePlatformConfiguration?: FargatePlatformConfiguration;
+
+  /**
+   * The IAM role that AWS Batch can assume.
```

Comment:
Mention that this is required when running on fargate.

Reply:
Done